package-build/build-package.sh

```diff
@@ -17,15 +17,10 @@ tar -xzf $ARCHIVE_NAME
 
 cd $LIB_DIR
 
-echo -e "\n\nCalling debmake\n"
```

Comment:
Any reason for removing this `echo` call?

Reply:
To be clear I'm proposing removing the debmake call entirely, and instead including the files that are needed that debmake might provide stubs for (and excluding the unneeded files).    So removing the echo is just a part of that.